hs-bindgen/src-internal/HsBindgen/Language/C/Name.hs

```diff
@@ -25,3 +31,12 @@ data Namespace =
   | NamespaceFunction
   deriving stock (Show, Eq, Ord, Generic)
 
+instance PrettyTrace Namespace where
+  prettyTrace = \case
+    NamespaceTypedef  -> "typedef"
+    NamespaceStruct   -> "struct"
+    NamespaceUnion    -> "union"
+    NamespaceEnum     -> "enum"
+    -- I made these two up.
+    NamespaceMacro    -> "C macro"
```

Comment:
I guess the "C" part can be omitted here?

Reply:
Also, @TravisCardwell might have some thoughts here, as we were working towards not having this `Namespace` at all. Not sure what we should do.